### PR TITLE
test: use linuxdeepin pr 61 test doc check

### PR DESCRIPTION
**Pull request by tsic404**
use linuxdeepin/dtkcore pr 61 test doc check

### CODE REVIEW

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
rehceck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
Here is the body for the review.

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
Here is the body for the review.

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
Here is the body for the review.

**Review by Zeno-sole — COMMENTED** [review]
This is close to perfect! Please address the suggested inline change.

**Review by Zeno-sole — COMMENTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/log/CuteLogger_global.h [hunk 1]
```diff
@@ -30,4 +30,8 @@
 #endif
 #endif
```
Comment (Zeno-sole) [inline src/log/CuteLogger_global.h:32]: test

**Review by Zeno-sole — COMMENTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/log/CuteLogger_global.h [hunk 2]
```diff
@@ -30,4 +30,8 @@
...
 
+#ifndef CUTELOGGER_DEFAULT_LOG_FORMAT
```
Comment (Zeno-sole) [inline src/log/CuteLogger_global.h:33]: define CUTELOGGER_DEFAULT_LOG_FORMAT is not documented!

**Review by Zeno-sole — COMMENTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/log/CuteLogger_global.h [hunk 3]
```diff
@@ -30,4 +30,8 @@
...
+#define CUTELOGGER_DEFAULT_LOG_FORMAT QLatin1String("%{time}{yyyy-MM-dd, HH:mm:ss.zzz} [%{type:-7}] [%{file:-20} %{function:-35} %{line}] %{message}\n")
```
Comment (Zeno-sole) [inline src/log/CuteLogger_global.h:34]: define CUTELOGGER_DEFAULT_LOG_FORMAT is not documented!

**Review by Zeno-sole — COMMENTED** [review]
This is close to perfect! Please address the suggested inline change.

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

File: src/log/CuteLogger_global.h (hunk 3 above)
Comment (Zeno-sole) [inline src/log/CuteLogger_global.h:34]: define CUTELOGGER_DEFAULT_LOG_FORMAT is not documented!

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
There is still something that need to be documented!

File: src/log/CuteLogger_global.h (hunk 3 above)
Comment (Zeno-sole) [inline src/log/CuteLogger_global.h:34]: define CUTELOGGER_DEFAULT_LOG_FORMAT is not documented!

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
There is still something that need to be documented!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
All has documented well!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
You're A Good Friend!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
You're So Kind!

**Review by Zeno-sole — COMMENTED** [review]
Great Answer!

**Review by Zeno-sole — COMMENTED** [review]
You're Sharp!

**Review by Zeno-sole — COMMENTED** [review]
You Made It Happen!

**Review by Zeno-sole — COMMENTED** [review]
You're Super!

**Review by Zeno-sole — COMMENTED** [review]
You're A Good Friend!

**Review by Zeno-sole — COMMENTED** [review]
You're A Champ!

**Review by Zeno-sole — COMMENTED** [review]
You're A Shining Star!

File: src/log/CuteLogger_global.h (hunk 2 above)
Comment (Zeno-sole) [inline src/log/CuteLogger_global.h:33]: define CUTELOGGER_DEFAULT_LOG_FORMAT is not documented!

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
There is still something that need to be documented!

File: src/log/CuteLogger_global.h (hunk 3 above)
Comment (Zeno-sole) [inline src/log/CuteLogger_global.h:34]: define CUTELOGGER_DEFAULT_LOG_FORMAT is not documented!

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
There is still something that need to be documented!

File: src/log/CuteLogger_global.h (hunk 3 above)
Comment (Zeno-sole) [inline src/log/CuteLogger_global.h:34]: define CUTELOGGER_DEFAULT_LOG_FORMAT is not documented!

**Review by Zeno-sole — CHANGES_REQUESTED** [review]
There is still something that need to be documented, Come on it!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
You're An Angel!

**Comment by tsic404** [comment]
recheck

**Review by Zeno-sole — COMMENTED** [review]
5 Star Work!

**Comment by tsic404** [comment]
reche

**Pull request closed without merging**